packages/react/src/ReactAvoidThisRender.js

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
Just put this in ReactHooks where all the other hooks are. Also reuse the resolveDispatcher. I plan on removing the nice error message in prod anyway and let it fail with a "null" exception because it adds an unfortunate overhead in the Hook call and keeps it from inlining.

There are some that are their own file but I plan on just inlining everything. Our isomorphic file is embarrassingly large.